dependencies/download_torch.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020-2022 OpenDR European Project
```

Comment:
ασδαδα

Reply:
test

Reply:
test comment

Reply:
This script is specifically designed for the installation of the C and CPP libraries of the Tοrch and torch vision packages. It will not affect any other components or functionality on the system. Is it better that if the user encounter any warnings during the installation process, the script whould may choose to download the CPU vesrions of these libraries instead?

Reply:
If when running the script you have defined that you want a GPU installation, and it fails along the way it would be unexpected to install CPU version instead, among various other reasons. 

I would suggest changing the warnings to exceptions that stop execution when it fails, providing a relative message.